src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -439,29 +463,30 @@ DOCKER_OPTS=" -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock"
 http://localhost:2375/info
 ----
 
-Having verified that your Docker daemon is listening, run the Che container with the with `DOCKER_HOST` environment variable set to the IP address of `docker0` or `eth0` network interface. If `docker0` is running on 1.1.1.1 then:
+After you have verified that your Docker daemon is listening, run the Che container with `DOCKER_HOST` environment variable set to the IP address of `docker0` or `eth0` network interface. If `docker0` is running on 1.1.1.1:
```

Comment:
* the `bla` variable
* the `bla` network interface
* last sentence missing verb